thesis-de/content-de.tex

```diff
@@ -769,6 +769,19 @@ \subsection{Frontends}
 
 \subsubsection{Definition eines Graphen}
 \label{definition_eines_graphen}
+Auf der Weboberfläche werden dem User oder dem Administrator einzelne Graphen,
```

Comment:
direkt zitieren würd ich nicht. Buch lesen, in eigenen worten wiedergeben und dann quelle angeben. wenn du mir die ISBN gibst kann ich das in biblatex einbauen.